examples/gallery/external/deck.gl.ipynb

```diff
@@ -111,7 +111,7 @@
     "\n",
     "widget.jslink(deckgl, code={'active': update_city});\n",
     "\n",
-    "pn.Column(widget, deckgl)"
+    "pn.Column(widget, deckgl).servable()"
```

Comment:
jslink based examples do not need servable.

```suggestion
    "pn.Column(widget, deckgl)"
```

Reply:
Ok, but it might be good to emphasize in that notebook how to make a pure HTML exported version (if not using panel serve as in other examples).  Maybe should .servable() still be used, for use with `panel html` (which currently isn't supported), so that someone can write a notebook and then serve it as HTML in the same way as `panel serve` does for a live Python server?